automation/devops_automation_infra/utils/kubectl.py

```diff
@@ -0,0 +1,35 @@
+import base64
+import json
+import os
+import logging
+from kubernetes.client import ApiException
+import kubernetes
+
+def create_generic_secret(client, name, data, namepace='default', type='Opaque'):
+    v1 = kubernetes.client.CoreV1Api(client)
+    sec = kubernetes.client.V1Secret()
+    sec.metadata = kubernetes.client.V1ObjectMeta(name=name)
+    sec.type = type
+    sec.data = data
+    v1.create_namespaced_secret(namespace=namepace, body=sec)
+
+
+def create_image_pull_secret(client, config_path=None):
+    if not config_path:
+        config_path = f"{os.environ['userhome']}/.docker/config.json"
+    with open(config_path, 'rb') as f:
+        data = {'.dockerconfigjson': base64.b64encode(f.read()).decode()}
```

Comment:
NIT:
can you please rename it to docker_images_creds .. otherwise it is not clear what is this in this file and why do we add it as secret